rest_log/components/service.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# @author Guewen Baconnier <guewen.baconnier@camptocamp.com>
+# @author Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import json
+import traceback
+
+from werkzeug.urls import url_encode, url_join
+
+from odoo import exceptions, registry
+from odoo.http import request
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class RESTServiceDispatchException(Exception):
```

Comment:
@simahawk I would prefer to have the exeception classes defined into `exceptions.py`